nxdrive/gui/api.py

```diff
@@ -263,6 +264,22 @@ def set_direct_edit_auto_lock(self, value: bool) -> None:
     def get_direct_edit_auto_lock(self) -> bool:
         return self._manager.get_direct_edit_auto_lock()
 
+    @pyqtSlot(result=list)
+    def get_features_list(self) -> List[List[str]]:
+        result = []
+        for key, value in Feature.__dict__.items():
```

Comment:
```suggestion
        for feature, state in vars(Feature):
```
- Use `vars(Feature)` to iterate.
- Use better names for variables.